ClinGen/clincoded
# Connects to #1984: Updates link for MaxEntScan

**Steps to test #1984**

1. Enter a new variant or select a variant to curate.
2. Proceed to the **Variant Type** tab in the evidence view
3. In the **Splice Site Predictors** panel in the **Missense** tab, click the **Analyze using MaxEntScan** link
4. Expect a new tab to open with the following link: http://hollywood.mit.edu/burgelab/maxent/Xmaxentscan_scoreseq.html
5. Repeat step 3 and 4 for the **Silent & Intron** tab
